FreeRTOS/FreeRTOS-Kernel-Partner-Supported-Ports
# Add new port layer for CCRH U2x and GHS U2x

Description
-----------
Add port files for CCRH and GHS cores for Renesas U2x devices.

Test Steps
-----------
Conducted the Demo project based on [Template for ThirdParty](https://github.com/FreeRTOS/FreeRTOS/blob/main/FreeRTOS/Demo/ThirdParty/Template/README.md). Refer to [FreeRTOS-Partner-Supported-Demos](https://github.com/renesas/FreeRTOS-Partner-Supported-Demos/tree/u2x_port)

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
